### PR TITLE
metrics: Add memory measurement inside the container.

### LINKED ISSUE #287: Add memory measurement inside the container
This will measure memory free, memory available and total memory inside the container with /proc/meminfo.

### PR DESCRIPTION
**Pull request by GabyCT**
This will measure memory free, memory available and total memory
inside the container with /proc/meminfo.

Fixes #287

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: metrics/density/memory_usage_inside_container.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+#  Description of the test:
+#  This test launches an alpine container and inside
+#  memory free, memory available and total memory
+#  is measured by using.
```
Comment (chavafg) [inline metrics/density/memory_usage_inside_container.sh]: `using information from /proc/meminfo`?

File: metrics/density/memory_usage_inside_container.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+	metrics_json_init
+
+	docker run --rm --runtime=$RUNTIME $IMAGE $CMD > $TMP_FILE
+	local output=$(cat $TMP_FILE)
+
+	# Save configuration
+	metrics_json_start_array
+
+	local memtotal=$(echo "$output" | grep -w "MemTotal" | cut -d ':' -f 2 | sed "s/^[ \t]*//" | cut -d ' ' -f1)
```
Comment (chavafg) [inline metrics/density/memory_usage_inside_container.sh]: You can use awk here:
`echo $output | awk '/MemTotal/ {print $2}' `

File: metrics/density/memory_usage_inside_container.sh [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	local units_memtotal=$(echo "$output" | grep -w "MemTotal" | cut -d ':' -f 2 | sed "s/^[ \t]*//"  | cut -d ' ' -f2)
```
Comment (chavafg) [inline metrics/density/memory_usage_inside_container.sh]: `echo $output | awk '/MemTotal/ {print $2}`

And same for the below commands

**Comment by GabyCT** [comment]
@chavafg thanks for the feedback, changes applied

**Comment by chavafg** [comment]
lgtm

**Comment by jodh-intel** [comment]
lgtm

**Pull request merged**